kafka/metadata.csv

```diff
@@ -6,15 +6,18 @@ kafka.net.processor.avg.idle.pct.rate,gauge,10,fraction,,Average fraction of tim
 kafka.messages_in.rate,gauge,10,message,,Incoming message rate.,0,kafka,messages in,cpu
 kafka.request.channel.queue.size,gauge,10,request,,Number of queued requests.,-1,kafka,req channel queue size,
 kafka.request.fetch_consumer.time.avg,gauge,10,millisecond,,Total time in ms to serve the specified request.,0,kafka,req fetch time avg,memory
+kafka.request.fetch_consumer.rate,gauge,10,request,,Number of fetch consumer requests.,0,kafka,req fetch consumer,
```

Comment:
Nit. Given these are rate, should the description say per second?

Reply:
That's true, I originally just kept it in the same format as the other existing rate descriptions.